ranges.md

```diff
@@ -0,0 +1,384 @@
+---
+title: Semantic Version Ranges
+---
+
+Semantic Version Range Selectors
+================================
+
+Summary
+-------
+
+A SemVer `Range` is a set of `Comparators` that specify a set of SemVer
+Versions.
+
+A Comparator is composed of an `Operator` and a `Version`.  The set
```

Comment:
Is `!=` also a primitive operator? I don't think it's possible to implement through the other operators